main/io.home-assistant/door.js

```diff
@@ -0,0 +1,38 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of io.home-assistant
+//
+// Copyright 2021
+//
+// See LICENSE for details
+"use strict";
+
+const HomeAssistantDevice = require('./base');
+
+module.exports = class HomeAssistantDoor extends HomeAssistantDevice {
+    async get_state() {
+        if (this.domain === 'sensor') {
```

Comment:
Does this class need a constructor setting `this.domain` too? Or should we factor the code that sets `this.domain` into the base HomeAssistantDevice ?

Reply:
wrong file, now updated new one